app/services/hyrax/collections/collection_member_service.rb

```diff
@@ -148,9 +148,20 @@ def remove_member(collection_id:, member:, user:)
           return member unless member?(collection_id: collection_id, member: member)
           member.member_of_collection_ids.delete(collection_id)
           member = Hyrax.persister.save(resource: member)
-          Hyrax.publisher.publish('object.metadata.updated', object: member, user: user)
+          publish_metadata_updated(member, user)
           member
         end
+
+        private
+
+        def publish_metadata_updated(member, user)
+          case member
```

Comment:
i think this should be based on `member.collection?` to conform to the engine's commitments about typing for the PCDM classes?

Reply:
That's much better.  Thanks for the suggestion.